ocean_lib/ocean/util.py

```diff
@@ -55,8 +55,6 @@ def get_infura_url(infura_id, network):
     if conn_type == "websocket":
         return f"wss://{network}.infura.io/ws/v3/{infura_id}"
 
-    raise AssertionError(f"Unknown connection type {conn_type}")
```

Comment:
why was this removed?

Reply:
I saw that by default the connection type was set to `http`, when the type is neither `http`, neither `websocket`. But I will revert it.